tests/test_deprecated.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2020 - 2021 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import unittest
+
+from monai.utils import DeprecatedError, deprecated, deprecated_arg
+
+
+class TestDeprecated(unittest.TestCase):
+    def setUp(self):
+        self.test_version = "0.5.3+96.g1fa03c2.dirty"
```

Comment:
I tested cases when test_version > next_version > prev_version, some of the `DeprecatedError` were missing